tezos/api/src/ffi.rs

```diff
@@ -356,63 +354,61 @@ pub struct ForkingTestchainData {
     pub test_chain_id: ChainId,
 }
 
+/// Represents a trace of errors produced by Tezos.
+///
+/// `head_error_id` is the id of the main error in the trace, useful for mapping into a Rust error.
+/// `trace_json` is json of the trace.
+pub struct TezosErrorTrace {
+    pub head_error_id: String,
+    pub trace_json: String,
+}
+
 #[derive(Serialize, Deserialize, Debug, Fail, PartialEq)]
 pub enum CallError {
-    #[fail(display = "Failed to call - message: {:?}!", parsed_error_message)]
+    #[fail(display = "Failed to call - message: {:?}!", trace_message)]
     FailedToCall {
-        parsed_error_message: Option<String>,
+        error_id: String,
```

Comment:
@tizoc 
what about error_id, could be added to display?

Reply:
It is the same as the id that shows up in the first error in `trace_message` (which is a JSON string), but yes, it is better to have it there anyway, will add.